CRM/Utils/Date.php

```diff
@@ -369,7 +369,7 @@ public static function customFormat($dateString, $format = NULL, $dateParts = NU
 
         $hour24 = (int) substr($dateString, 11, 2);
         $minute = (int) substr($dateString, 14, 2);
-        $second = (int) substr($dateString, 16, 2);
+        $second = (int) substr($dateString, 17, 2);
```

Comment:
this was only just added in the rc - if it is wrong then we should change this line in the rc

Reply:
(note the passive voice - had it been correct I could have said 'I added it' but since it seems It might be wrong.....)

Reply:
You could have said 'it was not noticed' :-)

Reply:
Hehe. Yes one could have said that.